docs/user-guides/1-finding-xrpackages.md

```diff
@@ -0,0 +1,40 @@
+---
+id: finding-xrpackages
+title: Finding Applications (XRPackages)
+---
+
+The Webaverse revolves around "applications" packaged into "XRPackages" (XRPKs).
+
+This guide explains how you can find XRPKs that other users have created and published.
+
+## Browsing for XRPackages
+
+You can easily browse, download, inspect, and modify XRPackages via <a href="https://xrpackage.org/browse.html" target="_blank" rel="noopener noreferrer">the XRPackage site</a> &mdash; all through your browser!
```

Comment:
I would avoid wishy-washy terms like "easy" in the documentation. The docs should be a place to inform the user and give them answers, not to sell them on how good the solutions are.